0.13/hoarder/src/main/scala/org/romanowski/hoarder/actions/CachedCI.scala

```diff
@@ -1,22 +1,24 @@
 package org.romanowski.hoarder.actions
 
-import java.nio.file.{Files, Path, Paths}
+import java.nio.file.Path
+import java.nio.file.Paths
 
 import org.romanowski.HoarderSettings._
 import org.romanowski.hoarder.core.HoarderEngine
 import sbt.Def._
 import sbt.Keys._
 import sbt._
 
-object CachedCI {
+object CachedCI extends HoarderEngine {
 
   trait Setup {
```

Comment:
@chipsenkbeil Is this interface enough for your planned caches? Idea here is that this single object is responsible for whole cache management for your CI.